From 4b7137ecff803c88e31cdeec41dd2751feacd3a8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timoth=C3=A9e=20Ravier?= <tim@siosm.fr>
Date: Fri, 15 Mar 2024 11:24:49 +0100
Subject: [PATCH] aboutdata: Update website

This is used to register the name on the bus and does not work well with
Flatpak.

See: https://github.com/flathub/org.kde.atlantik/pull/15
---
 client/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/main.cpp b/client/main.cpp
index 59ee5e9..ad6e19b 100644
--- a/client/main.cpp
+++ b/client/main.cpp
@@ -40,7 +40,7 @@ int main(int argc, char *argv[])
 		KAboutLicense::GPL,
 		i18n("(c) 1998-2004 Rob Kaper"),
 		i18n("KDE client for playing Monopoly-like games on the monopd network."),
-		QStringLiteral("http://unixcode.org/atlantik/")
+		QStringLiteral("https://apps.kde.org/atlantik/")
 		);
 	aboutData.setDesktopFileName(QStringLiteral("org.kde.atlantik"));
 
-- 
2.44.0

